boards/nxp/twr_ke18f/Kconfig.twr_ke18f

```diff
@@ -4,3 +4,4 @@
 config BOARD_TWR_KE18F
 	select SOC_MKE18F16
 	select SOC_PART_NUMBER_MKE18F512VLL16
+	select PINCTRL
```

Comment:
nor here

Reply:
https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/nxp/twr_ke18f/pinmux.c

Reply:
this one is acceptable

Reply:
It needs to go into `Kconfig`, not `Kconfig.<board_name>`

Reply:
Thank you, appreciate the help!